lib/java/akka-wrapper/src/main/java/module-info.java

```diff
@@ -0,0 +1,156 @@
+module org.enso.akka.wrapper {
+  requires protobuf.java;
+  requires scala.library;
+  requires org.reactivestreams;
+  // For sun.misc.Unsafe - from akka.util.Unsafe
+  requires jdk.unsupported;
+  requires org.slf4j;
+
+  // akka-actor
+  exports akka;
+  exports akka.actor;
+  exports akka.actor.dungeon;
+  exports akka.actor.setup;
+  exports akka.annotation;
+  exports akka.compat;
+  exports akka.dispatch;
+  exports akka.dispatch.affinity;
+  exports akka.dispatch.internal;
+  exports akka.dispatch.sysmsg;
+  exports akka.event;
+  exports akka.event.japi;
+  exports akka.event.jul;
+  exports akka.io;
+  exports akka.io.dns;
+  exports akka.io.dns.internal;
+  exports akka.japi;
+  exports akka.japi.function;
+  exports akka.japi.pf;
+  exports akka.japi.tuple;
+  exports akka.pattern;
+  exports akka.pattern.extended;
+  exports akka.pattern.internal;
+  exports akka.routing;
+  exports akka.serialization;
+  exports akka.util;
+  exports akka.util.ccompat;
+
+  // akka-actor-typed
+  exports akka.actor.typed;
```

Comment:
Hasn't `akka...typed` been removed in #10905?

Reply:
Yes, `exports akka.actor.typed.*` has been already removed in the latest revision on this PR.